### PR TITLE
1451 alignment off in the create post module post in

### PR DESCRIPTION
**Pull request by dcetlin**
I tried to resolve this with relative css; however, I think it requires hardcoded adjustments. I create a css class that only applies when the tag list is empty because the view behavior is different.

Screenshot 2023-10-03 at 12 22 48 PM
Screenshot 2023-10-03 at 12 22 52 PM
Screenshot 2023-10-03 at 12 22 56 PM
Screenshot 2023-10-03 at 12 23 02 PM

### CODE REVIEW

File: src/components/GroupsSelector/GroupsSelector.scss [hunk 1]
```diff
@@ -60,6 +63,7 @@
   /* prevent autoresize overflowing the container */
   max-width: 100%;
   line-height: 1;
+  /* Adjusts search bar position height*/
```
Comment (tibetsprague) [inline src/components/GroupsSelector/GroupsSelector.scss:66]: add a space before the */ just for aesthetics :)

File: src/components/GroupsSelector/GroupsSelector.scss [hunk 2]
```diff
@@ -87,6 +91,13 @@
   line-height: inherit;
 }
 
+[class*='tags-empty'] {
+  /* Adjusts search bar position height when no tagged groups are selected*/
```
Comment (tibetsprague) [inline src/components/GroupsSelector/GroupsSelector.scss:95]: space before */

File: src/components/GroupsSelector/GroupsSelector.scss [hunk 3]
```diff
@@ -96,7 +107,7 @@
 }
 
 .suggestions {
-  position: absolute;
+  position:absolute;
```
Comment (tibetsprague) [inline src/components/GroupsSelector/GroupsSelector.scss:110]: should be a space after the :

File: src/components/GroupsSelector/GroupsSelector.scss [hunk 4]
```diff
@@ -87,6 +91,13 @@
   line-height: inherit;
 }
 
+[class*='tags-empty'] {
+  /* Adjusts search bar position height when no tagged groups are selected*/
+  position: absolute;
+  z-index: 999;
+  transform: translateY(-19px);
```
Comment (tibetsprague) [inline src/components/GroupsSelector/GroupsSelector.scss:98]: why transform instead of just adjusting the top position?

**Review by tibetsprague — COMMENTED** [review]
looks good! A few small comments

**Comment by dcetlin** [comment]
@tibetsprague just updated. Thanks for the review.

**Comment by dcetlin** [comment]
Oh yikes looks like I somehow reverted a lot of files when I git pulled. Gonna try to figure out how to undo

**Comment by dcetlin** [comment]
Just did this @thomasgwatson 
https://github.com/Hylozoic/hylo-evo/pull/1557

I realized I rebased on master (default habit of mine) instead of dev.

**Pull request closed without merging**